katas/content/single_qubit_gates/index.md

```diff
@@ -365,6 +365,65 @@ All the basic gates we will be covering in this kata are part of the <a href="ht
     "solutionPath": "./sign_flip_on_zero/solution.md"
 })
 
+@[section]({
+    "id": "identity_gate",
+    "title": "Identity Gate"
+})
+
+The identity gate is mostly here for completeness, at least for now. It will come in handy when dealing with multi-qubit systems and multi-qubit gates. It is represented by the identity matrix, and does not affect the state of the qubit.
+  
+<table>
+<tr>
+<th>Gate</th>
+<th>Matrix</th>
+<th>Ket-Bra</th>
+<th>Applying to $|\psi\rangle = \alpha|0\rangle + \beta|1\rangle$</th>
+<th>Q# Documentation</th>
+</tr>
+<tr>
+<td>$I$</td>
+<td>$\begin{bmatrix} 1 & 0 \\ 0 & 1 \end{bmatrix}$</td>
+<td>$|0\rangle\langle0| + |1\rangle\langle1|$</td>
+<td>$I|\psi\rangle = |\psi\rangle$</td>
+<td><a href=\"https://docs.microsoft.com/qsharp/api/qsharp/microsoft.quantum.intrinsic.i\">I</a></td>
+</tr>
+</table>
+
+@[section]({
+    "id": "hadamard_gate",
+    "title": "Hadamard Gate"
+})
+
+The **Hadamard** gate is an extremely important quantum gate. Unlike the previous gates, applying the Hadamard gate to a qubit in a computational basis state puts that qubit into a superposition.
+Like the Pauli gates, the Hadamard gate is self-adjoint.
+
+<table>
+<tr>
+<th>Gate</th>
+<th>Matrix</th>
+<th>Ket-Bra</th>
+<th>Applying to $|\psi\rangle = \alpha|0\rangle + \beta|1\rangle$</th>
+<th>Applying to basis states</th>
+<th>Q# Documentation</th>
+</tr>
+<tr>
+<td>$H$</td>
+<td>$\begin{bmatrix} \frac{1}{\sqrt{2}} & \frac{1}{\sqrt{2}} \\ \frac{1}{\sqrt{2}} & -\frac{1}{\sqrt{2}} \end{bmatrix} = \frac{1}{\sqrt{2}}\begin{bmatrix} 1 & 1 \\ 1 & -1 \end{bmatrix}$</td>
+<td>$|0\rangle\langle+| + |1\rangle\langle-|$</td>
+<td>$H|\psi\rangle = \alpha|+\rangle + \beta|-\rangle = \frac{\alpha + \beta}{\sqrt{2}}|0\rangle + \frac{\alpha - \beta}{\sqrt{2}}|1\rangle$</td>
+<td>$H|0\rangle = |+\rangle \\
+H|1\rangle = |-\rangle \\
+H|+\rangle = |0\rangle \\
+H|-\rangle = |1\rangle \\
+H|i\rangle = e^{i\pi/4}|-i\rangle \\
+H|-i\rangle = e^{-i\pi/4}|i\rangle $</td>
+<td><a href=\"https://docs.microsoft.com/qsharp/api/qsharp/microsoft.quantum.intrinsic.h\">H</a></td>
+</tr>
+</table>
+
+> As a reminder, $e^{i\pi/4} = \frac{1}{\sqrt2} (1 + i)$ and $e^{-i\pi/4} = \frac{1}{\sqrt2} (1 - i)$.
+> If you need a refresher on calculating expressions like $e^{i\theta}$, you should review the section on [complex exponentiation](../ComplexArithmetic/ComplexArithmetic.ipynb#Imaginary-Exponents).
```

Comment:
Does this link work?

Reply:
It won't, since we don't have .ipynb files on the site. We can use a link to Wikipedia or another resource like that for now, and revisit this after we port ComplexArithmetic tutorial